src/Domain/FunctionsOfTime/QuaternionHelpers.cpp

```diff
@@ -0,0 +1,37 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Domain/FunctionsOfTime/QuaternionHelpers.hpp"
+
+#include "Utilities/ErrorHandling/Assert.hpp"
```

Comment:
Please also include DataVector, boost, gsl (always include everything you use in the file).